src/custom/FlipCard/FlipCard.tsx

```diff
@@ -1,83 +1,94 @@
+import { styled } from '@mui/material';
 import React from 'react';
+import { BUTTON_MODAL_DARK, WHITE } from '../../theme/colors/colors';
 
-/**
- * Wrapper component for flip cards.
- *
- * @param {Object} props - The component props.
- * @param {Object} props.frontComponents - The components of the card front.
- * @param {string} props.backComponents - The components of the card back.
- * @param {Boolean} props.disableFlip - The card can be flip or not.
- *
- */
+export type FlipCardProps = {
+  duration?: number;
+  onClick?: () => void;
+  onShow?: () => void;
+  children: [React.ReactNode, React.ReactNode];
+};
 
-export interface FlipCardProps {
-  frontComponents: JSX.Element;
-  backComponents?: JSX.Element;
-  disableFlip?: boolean;
+function GetChild(children: [React.ReactNode, React.ReactNode], key: number) {
+  if (!children) throw Error('FlipCard requires exactly two child components');
+  if (children.length != 2) throw Error('FlipCard requires exactly two child components');
+
+  return children[key];
 }
 
-function FlipCard({ frontComponents, backComponents, disableFlip }: FlipCardProps): JSX.Element {
-  const [isFlipped, setIsFlipped] = React.useState(false);
+const Card = styled('div')(({ theme }) => ({
+  height: '100%',
+  backgroundColor: 'transparent',
+  perspective: theme.spacing(125)
+}));
+
+const InnerCard = styled('div')(({ theme }) => ({
+  padding: theme.spacing(2),
+  borderRadius: theme.spacing(1),
+  transformStyle: 'preserve-3d',
+  boxShadow: '0 4px 8px 0 rgba(0,0,0,0.2)',
+  backgroundColor: theme.palette.mode === 'dark' ? BUTTON_MODAL_DARK : WHITE,
```

Comment:
The colors have been chosen in accordance with the current Flippable cards implemented in Meshery playground.
The theme.pallete.secondary.elevatedComponents in the original meshery implementation was giving errors and hence this change was made.